runner/runner_test.go

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2022 Renesas Electronics Corporation.
+// Copyright (C) 2022 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runner_test
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/aoscloud/aos_common/aoserrors"
+	"github.com/aoscloud/aos_servicemanager/runner"
+	"github.com/coreos/go-systemd/v22/dbus"
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	log.SetFormatter(&log.TextFormatter{
+		DisableTimestamp: false,
+		TimestampFormat:  "2006-01-02 15:04:05.000",
+		FullTimestamp:    true,
+	})
+	log.SetLevel(log.DebugLevel)
+	log.SetOutput(os.Stdout)
+}
+
+/***********************************************************************************************************************
+ * Consts
+ **********************************************************************************************************************/
+
+const serviceTemplate = `[Unit]
+Description=AOS Service
+After=network.target
+StartLimitIntervalSec=2
+StartLimitBurst=1
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/bin/sh /run/aos/runtime/%i/service.sh
+
+PIDFile=/run/aos/runtime/%i/.pid
+SuccessExitStatus=SIGKILL
+
+[Install]
+WantedBy=multi-user.target
+`
+
+const serviceContent = `
+#!/bin/bash
+echo "HELLO!!!!"
+sleep 10
+`
+const binFileName = "service.sh"
+
+const runtimeDir = "/run/aos/runtime"
+
+const (
+	failedState = "failed"
```

Comment:
Should be used from cloudprotocol consts.

Reply:
done